docs/extensions_dev.md

```diff
@@ -1,9 +1,12 @@
 # Extensions Developer Guide
 
-JupyterLab can be extended in two ways via:
+JupyterLab can be extended in three ways via:
 
 - **application plugins (top level):** Application plugins extend the
-  functionality of JupyterLab itself, and **this tutorial focuses on them.**
+  functionality of JupyterLab itself.
+- **mime renderer extension (top level):**  Mime Renderer extensions are 
+  a convenience for creating an extension that can render mime data and
+  potentially render files of a given type. 
 - document widget extensions (lower level): Document widget extensions extend
   the functionality of document widgets added to the application, and we cover
   them in the "Documents" tutorial.
```

Comment:
Note: The Documents tutorial previously mirrored this bullet-point list, so it should probably get an update too.